packages/boot/test/bootstrapTests/ibcClientMock.js

```diff
@@ -1,7 +1,7 @@
 /** @file Mock IBC Server */
 // @ts-check
 import { Far } from '@endo/far';
-import { V as E } from '@agoric/vat-data/vow.js';
+import { V as E } from '@agoric/vow/vat.js';
```

Comment:
Where is the `'@agoric/vow/vat.js'` file? I don't see it.

Reply:
> Where is the `'@agoric/vow/vat.js'` file? I don't see it.

It's in the base PR, #9321.

Reply:
Thx. Resolving.